praw/models/reddit/live.py

```diff
@@ -37,3 +59,96 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
 
     def _info_path(self):
         return API_PATH['liveabout'].format(id=self.id)
+
+    @property
```

Comment:
Does this not work properly from the `fullname` defined at:
https://github.com/praw-dev/praw/blob/a90b7a2ec07bfaf1f6dd1b8b06e0c338792fbca8/praw/models/reddit/base.py#L11